types/react-material-ui-form-validator/react-material-ui-form-validator-tests.tsx

```diff
@@ -6,7 +6,9 @@ import {
 } from 'react-material-ui-form-validator';
 
 class Test extends React.Component {
-    onSubmitted = (event: React.FormEventHandler) => {};
+    onSubmitted = (event: React.FormEvent) => {
+        alert(event.target);
```

Comment:
No UI Blocking code in test please

Reply:
@FrankBrullo Thank you for the review. Changed test code. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38699/commits/626687de53049af6a993a5b2eeb69fc7816c6fdb